string pose
float speed
---
byte status
